### PR TITLE
GitHubAuthentication2

### PR DESCRIPTION
**Pull request by MariaJose319**

### CODE REVIEW

**Review by danielgalvis98 — COMMENTED** [review]
Please check that the token you have configured on the secrets is the same you are using locally as environment variable (In case that tests work locally). Be sure that there are no extra spaces or any weird character

**Comment by MariaJose319** [comment]
> Please check that the token you have configured on the secrets is the same you are using locally as environment variable (In case that tests work locally). Be sure that there are no extra spaces or any weird character

I had a problem because I created an "environment secret" instead of a "repository secret".

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -20,3 +20,5 @@ jobs:
       env:
         ACCESS_TOKEN: ${{secrets.ACCESS_TOKEN}}
       run: npm test
+    - run: echo "Print all environment variables"
+    - run: env
```
Comment (kliver98) [inline .github/workflows/node.js.yml:24]: Delete this ones

**Review by kliver98 — COMMENTED** [review]
I think you could close this PR, it's not need it since the solution was not code but configuration inside github

**Pull request closed without merging**